lib/ecto_job/worker.ex

```diff
@@ -43,4 +50,13 @@ defmodule EctoJob.Worker do
     repo.query("SELECT pg_notify($1, $2)", [topic, payload])
     :ok
   end
+
+  # extract job from %{ :good_work => :nothing, "delete_job_9" 
+  # => %{ __meta__: "Ecto.Schema.Metadata<:deleted", attempt: 1,
+  defp extract_job(multi_response), do:
+     multi_response |> Map.to_list |> List.last |> elem(1)
```

Comment:
I'm not sure we need this function, but `Map.to_list` will produce a list sorted by the Map keys, so I don't think it is safe to assume what the contents of `List.last` will be.

Reply:
I liked your comments @mbuhot , thanks :)
Yes, I also agree that it's unsafe, but I couldn't find a better way to extract the job data structure,
I will think better